### DISCUSSION TITLE
Where is the emqx-management plugin to be installed?

### DISCUSSION THREAD

**Discussion by alexe100** [emqx/emqx#13782] [General]
Where is the qmxq-management plugin to be installed? 
tried to compile from github sources but:
===> Error building application emqx:
     No project builder is configured for type mix
(open-source version)
Thanks

**Comment by zmstone** [comment]
Hi @alexe100 
do you mean emqx-management plugin ?

about build failure, could you share your steps?

**Comment by alexe100** [comment]
Yes, it is that plugin. I need to check periodically who is connected (via HTTP API) or trap events inside emqx and log them to a database and build my own check API, but did not find a way to trap connect events yet (Event Notification?). So I am trying to rely on the emqx-management plugin .

Env: Ubuntu 22

sudo apt-get install libkrb5-dev
sudo apt-get install libsasl2-dev
https://s3.amazonaws.com/rebar3/rebar3 
git clone https://github.com/emqx/emqx-management.git
cd emqx-management
yarn
make
...
===> Error building application emqx:
     No project builder is configured for type mix

Thank you

**Reply by ieQu1** [reply]
Which version of EMQX are you using? 
If I remember correctly, this repo is only used for EMQX 4.2 and below. 
- EMQX 4.3 and above integrates this code into the monorepo, so there's no need to compile this as a plugin
- Because of that, code in https://github.com/emqx/emqx-management.git is not being updated and it's incompatible with up-to-date versions of EMQX.

**Comment by alexe100** [comment]
Version= 5.8

$sudo systemctl start emqx

$emqx_ctl plugins start emqx_management
{
  "result" : "not_ok",
  "name_vsn" : "emqx_management",
  "cause" : {
    "reason" : "plugin_not_found",
    "name_vsn" : "emqx_management",
    "msg" : "no_nodes_to_copy_plugin_from"
  },
  "action" : "ensure_started"
}

$emqx_ctl plugins list
{
  
}

I doesn't exist...

The idea is to fetch the list of connected users using HTTP API (or other approach)

**Reply by id** [reply]
@alexe100 you don't need management plugin in 5.8 or any other 5.x version. This is built in functionality.

**Comment by alexe100** [comment]
Great! 
http://xxxxx:18083/api-docs/index.html
Thank you!
